frontend/lib/parse-capi/index.ts

```diff
@@ -292,6 +292,10 @@ export const extractArticleMeta = (data: {}): CAPIType => {
         webPublicationDate,
         tags,
         sectionName,
+        webPublicationDateDisplay: getNonEmptyString(
```

Comment:
I'm guessing `webPublicationDateDisplay` is mandatory and we should always get this? `getNonEmptyString` is a strict check and if an article doesn't have this property the rendering engine will return a `500` error.

Reply:
That's correct @GHaberis. The value is always set.